@here/harp-materials/lib/SolidLineMaterial.ts

```diff
@@ -46,6 +46,7 @@ varying vec2 vSegment;
 varying float vResultLineWidth;
 varying vec3 vPosition;
 varying float vLength;
+varying float vUVDelta;
```

Comment:
Doyumentation please. Maybe you can also add some documentation for vLength

Reply:
Done.